services/data-info/src/data_info/services/rename.clj

```diff
@@ -39,14 +39,14 @@
   [{user :user} {source :source dest :dest}]
   (rename-path user source dest))
 
-(with-post-hook! #'do-rename (log-func "do-rename"))
+(with-post-hook! #'do-rename (dul/log-func "do-rename"))
 
 (with-pre-hook! #'do-rename
   (fn [params body]
-    (log-call "do-rename" params body)
+    (dul/log-call "do-rename" params body)
     (validate-map params {:user string?})
     (validate-map body {:source string? :dest string?})
```

Comment:
These param and body validations can be removed, since compojure-api will perform these validations before the request gets to this service function.
`validate-num-paths-under-folder` should still be called below.